pkg/reconciler/messaging/channel/channel.go

```diff
@@ -261,17 +263,19 @@ func (r *Reconciler) syncSubscribers(ctx context.Context, channel *v1alpha1.Chan
 	return nil
 }
 
-func (r *Reconciler) syncSubscribersStatus(ctx context.Context, channel *v1alpha1.Channel) error {
-	if channel.Status.SubscribableStatus == nil {
-		channel.Status.SubscribableStatus = &eventingduck.SubscribableStatus{
-			Subscribers: make([]eventingduckv1beta1.SubscriberStatus, 0),
-		}
-	} else if channel.Status.SubscribableStatus.Subscribers == nil {
-		channel.Status.SubscribableStatus.Subscribers = make([]eventingduckv1beta1.SubscriberStatus, 0)
-	}
+func (r *Reconciler) syncSubscribersStatus(ctx context.Context, channel *v1beta1.Channel) error {
+	//if equality.Semantic.DeepEqual(channel.Status.SubscribableStatus, eventingduckv1beta1.SubscribableStatus{}){
+	//	channel.Status.SubscribableStatus = eventingduckv1beta1.SubscribableStatus{
+	//		Subscribers: make([]eventingduckv1beta1.SubscriberStatus, 0),
+	//	}
+	//} else if channel.Status.SubscribableStatus.Subscribers == nil {
+	//	channel.Status.SubscribableStatus.Subscribers = make([]eventingduckv1beta1.SubscriberStatus, 0)
+	//}
+	if channel.Status.SubscribableStatus.Subscribers == nil {
```

Comment:
similar above. Since we use https://github.com/google/knative-gcp/blob/94e9331b180d315d058ea867b8e368096f807e80/pkg/client/injection/reconciler/messaging/v1beta1/channel/reconciler.go#L184,
nil or empty doesn't matter